sources/packages/web/src/components/layouts/aest/AESTApplicationSideBar.vue

```diff
@@ -28,11 +28,9 @@
     <v-list-item
```

Comment:
I see this as below:

<img width="1920" alt="image" src="https://github.com/user-attachments/assets/59e4ac76-0ba6-4a0b-95b8-fb6bab477c58">

The text gets cut at the end in the sidebar and there is no way to see it. Not sure if adding a tooltip would be a good idea here.

Reply:
Updated. It's now "Restriction Management".